node_lambnik/src/tiler/package.json

```diff
@@ -25,11 +25,12 @@
   "scripts": {
     "build-xml": "babel-node src/util/build-xml.js src/config/map-config.mml bin/map-config.xml",
     "deploy": "yarn transpile && claudia update ${LAMBDA_TIMEOUT:+--timeout ${LAMBDA_TIMEOUT}} ${LAMBDA_MEMORY:+--memory ${LAMBDA_MEMORY}} ${LAMBDA_SECURITY_GROUPS:+--security-group-ids ${LAMBDA_SECURITY_GROUPS}} ${LAMBDA_SUBNETS:+--subnet-ids ${LAMBDA_SUBNETS}}",
-    "deploy-new": "yarn transpile && claudia create --api-module bin/api --name tilegarden-dev-${USER} --region ${LAMBDA_REGION} --role ${LAMBDA_ROLE} ${LAMBDA_TIMEOUT:+--timeout ${LAMBDA_TIMEOUT}} ${LAMBDA_MEMORY:+--memory ${LAMBDA_MEMORY}} ${LAMBDA_SECURITY_GROUPS:+--security-group-ids ${LAMBDA_SECURITY_GROUPS}} ${LAMBDA_SUBNETS:+--subnet-ids ${LAMBDA_SUBNETS}}",
+    "deploy-new": "yarn transpile && claudia create --api-module bin/api --name ${PROJECT_NAME} --region ${LAMBDA_REGION} --role ${LAMBDA_ROLE} ${LAMBDA_TIMEOUT:+--timeout ${LAMBDA_TIMEOUT}} ${LAMBDA_MEMORY:+--memory ${LAMBDA_MEMORY}} ${LAMBDA_SECURITY_GROUPS:+--security-group-ids ${LAMBDA_SECURITY_GROUPS}} ${LAMBDA_SUBNETS:+--subnet-ids ${LAMBDA_SUBNETS}} && yarn parse-id",
     "dev": "nodemon -e js,mss,json,mml,mss --ignore bin/ --exec yarn local",
     "lint": "eslint src",
     "local": "yarn transpile && node --inspect=0.0.0.0:9229 node_modules/claudia-local-api/bin/claudia-local-api --api-module bin/api | bunyan -o short",
-    "transpile": "yarn build-xml && babel src -d bin --copy-files --source-maps inline"
+    "transpile": "yarn build-xml && babel src -d bin --copy-files --source-maps inline",
+    "parse-id": "jq -r '.api.id' claudia.json > .api-id"
```

Comment:
I don't love that we're adding another little artifact that's redundant with the existing `claudia.json`, but I do like that this minimizes the requirements for the host machine.  I think my first choice would be installing `jq` in the terraform container and giving it access to the claudia.json file, but this is fine, and easy to change later if it bothers us and/or we have an abundance of time.

Reply:
I agree with it being better to put `jq` in the terraform container, but I did it this way to avoid having to maintain another dockerfile that would have to change a lot of the behavior of the terraform image.